### PR TITLE
Fix code style error in docs

### PR DESCRIPTION
**Pull request by devSC**

### CODE REVIEW

File: docs/Endpoints.md [hunk 1]
```diff
@@ -123,14 +123,13 @@ Instead of modifying the request, you could simply log it, instead.
 
 ```swift
 let requestClosure = { (endpoint: Endpoint<GitHub>, done: MoyaProvider.RequestResultClosure) in
-    do {
-        var request = try endpoint.urlRequest()
-        // Modify the request however you like.
-        done(.success(request))
-    } catch {
-	done(.failure(MoyaError.underlying(error)))
```
Comment (SD10) [inline docs/Endpoints.md]: @devSC Thanks for this pull request, however, this is the correct code style for Moya `10.0`+.

File: docs_CN/Endpoints.md [hunk 2]
```diff
@@ -97,11 +97,12 @@ Sample responses 有下面的这些值:
 
 ```swift
 let requestClosure = { (endpoint: Endpoint<GitHub>, done: MoyaProvider.RequestResultClosure) in
-    var request = endpoint.urlRequest
-
-    // Modify the request however you like.
-
-    done(.success(request))
```
Comment (SD10) [inline docs_CN/Endpoints.md]: @devSC However, our Chinese docs here are invalid. We should be using `try` and `do/catch`. Care to update these?

File: docs/Endpoints.md (hunk 1 above)
Comment (devSC) [reply docs/Endpoints.md]: oh, i mistake.

File: docs_CN/Endpoints.md (hunk 2 above)
Comment (devSC) [reply docs_CN/Endpoints.md]: yes, i will open a new PR.

**Comment by SD10** [comment]
This is related to #1357. We need to update the Chinese docs to reflect the changes for the `urlRequest()` method of `Endpoint`.

**Pull request closed without merging**